src/pages/User.js

```diff
@@ -0,0 +1,44 @@
+import logo from '../CodeKingTrans.png';
+import Navbar from '../components/Navbar.js';
+import Footer from '../components/Footer.js';
+import List from '../components/List.js';
+import useFetch from '../hooks/useFetch';
+import '../App.css';
+
+function UserCard({ user }){
+    return (
+        <div className='userCard'>
+            <img alt='profile logo' src='https://cdn-icons-png.flaticon.com/512/12225/12225773.png'></img>
+            <h2>{user.firstName + ' ' + user.lastName}</h2>
+        </div>
+    );
+}
+
+
+function User() {
+    const { data: users, loading, error } = useFetch("https://disc-assignment-5-users-api.onrender.com/api/users");
```

Comment:
GREAT use of a custom hook to declutter your `User` component and hide away details of the fetch!